### PR TITLE
#7 - import info

### LINKED ISSUE #7: [Administrator panel] Providers synchronization
Administrator should be able to synchronize providers data. On the page it should be a button "Synchronize" and when the administrator clicks on it, a synchronization should be done. Also some automatic synchronization should be run for example every week.
Each synchronization should be shown in a table with information like:
- who runs it - user or it's automatic synchronization
- execution data
- status
  - when API fails, the application should still works, but status should be on red
  - when some action by administrator is required (for example when provider is in city that isn't in database)  status should be blue and information what action is required
  - if everything works, status should be green

### PR DESCRIPTION
**Pull request by vojcc**
This should close #7.

### CODE REVIEW

**Review by EwelinaLasowy — COMMENTED** [review]
1. Wrong pr title - as I understand from pr description, this pr is closing issue 7, not 52.
 
2. For import info, we should make another tab in navigation menu and there put button "Run importers" and table with information about running imports.

3. When I click "Run importers" in the table I see status "Success". When I take a look at, I notice that it's blinking - so it's suggesting that is in progress? If so, I will change text to "in progress" and text color to gray maybe?
Also I have to refresh the page to see that status changes to "warning" or something. See the video below to better understand what I mean.

https://github.com/blumilksoftware/escooters/assets/56546832/edb00ef1-48fa-49da-9f83-ca212cbbbc99

4. All the time I see "Warning". What should I do to see status "Success"?

image

5. Please add some pagination.

6. When import has status "Warning" we should show somehow that user can click on the status and see some information - maybe add some info icon or some arrow like it is in Telescope?

7. After import status shouldn't blinking - I mean that "Warning" status.

8.  If we have import in progress, maybe we should show somehow statuses of individual importers?

**Review by EwelinaLasowy — COMMENTED** [review]
1. For import info, we should make another tab in navigation menu and there put button "Run importers" and table with information about running imports. 

2. When I click "Run importers" button, I see status "Running"

image


It's great, but I have to refresh the page to see if status changes. What's more I always see "Warning" and I don't know how fix that to see "Success" status.

image

3. Please increase pagination to 15 results on page, because now we have 5 per page.

4. If we have import in progress, maybe we should show somehow statuses of individual importers?

5. Please make this dot and status in one line.

File: resources/js/Pages/Dashboard/Index.vue [hunk 1]
```diff
@@ -1,27 +1,47 @@
 <script setup>
 import AdminNavigation from '../../Shared/Layout/AdminNavigation.vue'
 import { usePage } from '@inertiajs/vue3'
+import Import from './Components/Import.vue'
+import PaginationInfo from '../../Shared/Components/PaginationInfo.vue'
```
Comment (EwelinaLasowy) [inline resources/js/Pages/Dashboard/Index.vue:5]: Please use `@` in import

**Review by EwelinaLasowy — COMMENTED** [review]
1. When I run command `php artisan migrate:fresh --seed` I see error:

image

2. Empty state is missing

image

3. We have some component for CTA button - please use that instead of styling another button.

**Comment by vojcc** [comment]
.env
```
MAPBOX_URL=https://api.mapbox.com/geocoding/v5
```

**Comment by EwelinaLasowy** [comment]
Please resolve conflicts.

**Review by MichalMyskow — APPROVED** [review]

**Review by EwelinaLasowy — APPROVED** [review]

**Review by AleksandraKozubal — APPROVED** [review]

**Pull request merged**